src/commands/codepush/lib/react-native-utils.ts

```diff
@@ -174,20 +174,24 @@ export function getReactNativeProjectAppVersion(versionSearchParams: VersionSear
     var appxManifestFileName: string = "Package.appxmanifest";
     try {
       var appxManifestContainingFolder: string = path.join("windows", projectName);
-      var appxManifestContents: string = fs.readFileSync(path.join(appxManifestContainingFolder, "Package.appxmanifest")).toString();
+      var appxManifestContents: string = fs.readFileSync(path.join(appxManifestContainingFolder, appxManifestFileName)).toString();
     } catch (err) {
       throw new Error(`Unable to find or read "${appxManifestFileName}" in the "${path.join("windows", projectName)}" folder.`);
     }
 
-    xml2js.parseString(appxManifestContents, (err: Error, parsedAppxManifest: any) => {
-      if (err) {
-        throw new Error(`Unable to parse the "${path.join(appxManifestContainingFolder, appxManifestFileName)}" file, it could be malformed.`);
-      }
-      try {
-        return parsedAppxManifest.Package.Identity[0]["$"].Version.match(/^\d+\.\d+\.\d+/)[0];
-      } catch (e) {
-        throw new Error(`Unable to parse the package version from the "${path.join(appxManifestContainingFolder, appxManifestFileName)}" file.`);
-      }
+    return new Promise<string>((resolve, reject) => {
+      xml2js.parseString(appxManifestContents, (err: Error, parsedAppxManifest: any) => {
+        if (err) {
+          throw new Error(`Unable to parse the "${path.join(appxManifestContainingFolder, appxManifestFileName)}" file, it could be malformed.`);
```

Comment:
Please replace it with 
```js
reject(new Error(`Unable to parse the "${path.join(appxManifestContainingFolder, appxManifestFileName)}" file, it could be malformed.`));
return;
```